assets/queries/terraform/alicloud/ros_stack_without_template/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.alicloud_ros_stack[name]
+
+	not resource.template_body
+	not resource.template_url
```

Comment:
```suggestion
    not common_lib.valid_key(resource, "template_body")
    not common_lib.valid_key(resource, "template_url")
```

Do not forget to import the common lib, please: `import data.generic.common as common_lib`

Please, also update in `kics\assets\queries\terraform\aws\stack_without_template\query.rego`